test/Makefile

```diff
@@ -0,0 +1,78 @@
+comma := ,
+colon := :
+space :=
+space +=
+
+define colorecho
+	@tput setaf 6
+	@echo $1
+	@tput sgr0
+endef
+
+define capture-output
+	script $1 $2 > /dev/null || cat $1
+endef
+
+ROOT ?= ..
+VIRTUALENV_ROOT := $(ROOT)/venv
+PATH := $(VIRTUALENV_ROOT)/bin:$(PATH)
```

Comment:
Same with PATH, VIRTUALENV_BINARY?
Also probably don't need to define VIRTUALENV_ROOT for a single use

Reply:
Fair

Reply:
PATH is the Unix PATH, can't change the name for that

Reply:
VIRTUALENV_ROOT is the same as in the root Makefile so don't really want to change that either...